src/main/com/fulcrologic/fulcro/mutations.cljc

```diff
@@ -331,11 +331,14 @@
   `env` - The env of the mutation
   `class` - A component class that represents the return type.  You may supply a fully-qualified symbol instead of the
   actual class, and this method will look up the class for you (useful to avoid circular references).
+  `opts` (optional):
+   - `query-params` - Optional parameters to add to the generated query
 
   Returns an update `env`, and is a valid return value from mutation remote sections."
   ([env class]
    (returning env class nil))
-  ([env class query-params]
+  ([env class {:keys [query-params]
```

Comment:
the problem here is that this a breaking change, users of query params will have unexpected results on upgrade. If we need more things here, better to add another argument or make a new function.

Reply:
@wilkerlucio this branch hasn't been released, so its not breaking yet!
Tony and I were discussing here: https://clojurians.slack.com/archives/C68M60S4F/p1619214744201500

Reply:
(For clarity, the `query-params` parameter was added after the last public release of fulcro)

Reply:
ah, ok, my bad then, sorry for taking conclusions looking only in the small

Reply:
No problem, I should/could have linked to the conversation in the PR.